pages/index.css

```diff
@@ -2,3 +2,11 @@
 @import url(../vendor/fonts/fonts.css);
 @import url(../blocks/page/page.css);
 @import url(../blocks/page/__container/page__container.css);
+
+@import url(../blocks/research/research.css);
+@import url(../blocks/research/__image/research__image.css);
+@import url(../blocks/research/__link/research__link.css);
+@import url(../blocks/research/__paragrah/research__paragraph.css);
```

Comment:
@Woolfs1423 Please check your import, right now it is not applying styles to your element. Since you renamed your folder in BEM Nested, it has to change in the import statement as well.